lib/components/Buttons/ButtonPrimary.tsx

```diff
@@ -14,26 +14,43 @@ interface Props {
   target?: string;
   isExternalHref?: boolean;
   disabled?: boolean;
+  secondary?: boolean;
 }
 
 export const ButtonPrimary: FC<Props> = ({ link: LinkElement, ...props }) => {
   let buttonStyle;
   if (props.variant === "gray") {
-    buttonStyle = cx(
-      styles.button_primary,
-      styles.button_primary_gray,
-      props.className
-    );
+    props.secondary
+      ? (buttonStyle = cx(
+          styles.button_secondary,
+          styles.button_secondary_gray,
+          props.className
+        ))
+      : (buttonStyle = cx(
+          styles.button_primary,
+          styles.button_primary_gray,
+          props.className
+        ));
   } else if (props.variant === "blueRounded") {
+    props.secondary
+      ? (buttonStyle = cx(
+          styles.button_secondary,
+          styles.button_secondary_blue,
+          props.className
+        ))
+      : (buttonStyle = cx(
+          styles.button_primary,
+          styles.button_primary_blue,
+          props.className
+        ));
+  } else {
     buttonStyle = cx(
-      styles.button_primary,
-      styles.button_primary_blue,
-      props.className
+      props.secondary ? styles.button_secondary : styles.button_primary,
+      props.className,
+      {
+        icon: props.variant === "icon",
+      }
     );
-  } else {
-    buttonStyle = cx(styles.button_primary, props.className, {
-      icon: props.variant === "icon",
-    });
```

Comment:
This whole block is very hard to read.

I would write the styles like this:

```TS
  const buttonStyle = cx(
    styles.buttonPrimary,
    {
      gray: props.variant === "gray",
    },
    {
      blueRounded: props.variant === "blueRounded",
    },
    {
      icon: props.variant === "icon",
    },
    props.className
  );
  ```

Reply:
And in the styles for `buttonPrimary`

```TS
export const buttonPrimary = css`
  ${buttonBase};
  background-color: var(--klima-green);

  &:active:not(:disabled) {
    transform: scale(0.95);
    opacity: 0.6;
  }
  &,
  &:hover:not(:disabled),
  &:visited {
    color: white; /* same in darkmode */
  }

  &:disabled {
    background-color: var(--surface-03);
    cursor: not-allowed;
  }

  &.icon {
    gap: 0.8rem;
  }

  &.gray {
    background-color: var(--surface-01);
    color: var(--font-02);

    &,
    &:hover,
    &:hover:not(:disabled),
    &:visited {
      color: var(--font-02); /* same in darkmode */
    }
  }

  &.blueRounded {
    background-color: var(--klima-blue);
    color: var(--surface-01);
    border-radius: 0.8rem;
    &,
    &:hover,
    &:hover:not(:disabled),
    &:visited {
      color: var(--surface-01); /* same in darkmode */
    }
  }
`;
```

Reply:
@ladytrekker @goatsOnDrugs lady is right we can avoid javascript and rely on cx, but actually you can condense it into one object.
```
 const buttonStyle = cx(
    styles.buttonPrimary,
    {
      gray: props.variant === "gray",
      blueRounded: props.variant === "blueRounded",
      icon: props.variant === "icon",
    },
    props.className
  );
```